docs/sources/developer/Manual-Dependencies-Fedora-Core.md

```diff
@@ -1,28 +1,28 @@
-# Building and Installing Dependencies on Fedora Core 
+# Manually Building and Installing Dependencies on Fedora Core 
```

Comment:
per chat remove the manual build instructions

Reply:
Done